modules/plugins/filetree/neo-tree/config.nix

```diff
@@ -21,7 +22,28 @@ in {
         setupModule = "neo-tree";
         inherit (cfg) setupOpts;
 
+        beforeAll =
+          optionalString (cfg.setupOpts.filesystem.hijack_netrw_behavior != "disabled")
+          # from https://github.com/nvim-neo-tree/neo-tree.nvim/discussions/1326
+          ''
+            vim.api.nvim_create_autocmd("BufEnter", {
+              group = vim.api.nvim_create_augroup("load_neo_tree", {}),
```

Comment:
We don't need the group, I never knew what it was for in the first place.

Reply:
Why would you need to delete it? Also it should be WhateverThisTypeOfCaseIsCalled instead of whatever_else_this_is_called.

Reply:
I'd rather keep the global namespace clean so users can use it however they want without consequences. and literally everyone uses it.

I counted 1 use of PascalCase and 2 uses of snake_case so I win :)

Reply:
mye, official docs have snake_case as well, I think I got PascalCase from LazyVim from something